docs/concepts/portal/dashboard.md

```diff
@@ -0,0 +1,14 @@
+---
+title: " Dashboard Page Reference"
+linkTitle: "Dashboard Page Reference"
+---
+
+
+The Dashboard gives you an unified and summarised view of the application you have deployed. Top view all of your application scroll the Dashboard page. If there are any persistent volumes in error this will be indicated on the dashboard as a red side line next to the application name and will also give you the number of affected volumes 
+
+Your deployed application can be one of the following types:
+* Replica - <add_description_here>
+* StatefulSet - <add_description_here>
+* Deployment - <add_description_here>
```

Comment:
A [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) provides a desired state for one or more sets of pods without any particular guarantees about order or uniqueness.

Reply:
Same